prac_06/box_layout_demo.py

```diff
@@ -3,15 +3,18 @@
 
 
 class BoxLayoutDemo(App):
+    """ This program shows how box layout format works, along with simple inputs and output labels"""
     def build(self):
         self.title = "Box Layout Demo"
         self.root = Builder.load_file('box_layout.kv')
         return self.root
 
+    """ Prints to the output label the name entered in the input box"""
```

Comment:
Docstrings should go inside (first line) functions and be written in imperative voice.